pkg/quota/evaluator/core/pods.go

```diff
@@ -269,18 +258,18 @@ func PodUsageFunc(obj runtime.Object, clock clock.Clock) (api.ResourceList, erro
 	if err != nil {
 		return api.ResourceList{}, err
 	}
-	// by convention, we do not quota pods that have reached end-of life
-	if !QuotaPod(pod, clock) {
-		return api.ResourceList{}, nil
+
+	// always quota the object count (even if the pod is end of life)
+	result := api.ResourceList{
+		podObjectCountName: *(resource.NewQuantity(1, resource.DecimalSI)),
```

Comment:
New resource name perhaps?  I could see that.  Counts include everything and there is a "special" count for "pods you probably meant to count".  If so, just a better comment above.

Reply:
added comment above to clarify.